Models/Pokemon.swift

```diff
@@ -10,31 +10,31 @@ import SwiftUI
 import CoreLocation
 
 struct Pokemon: Hashable, Codable, Identifiable {
-    var id: String
-    var number: Int
+    var id: Int
```

Comment:
Does id really need to be a string? I was using it, successfully, for por pokemon that share the same number but have different forms, like Raichu being`"026"` and Alolan Raichu being`"026a"`

Reply:
I just changed to Int, because the JSON data, the Id was an Int. It can be a string if the json Id's are strings.